src/interfaces/index.ts

```diff
@@ -0,0 +1,20 @@
+export interface INetwork {
+  name: string;
+  rpcUrl: string;
+  chainId: number;
+  payrollAddress: string;
+  routerAddress?: string;
```

Comment:
I think routerAddress shouldn't be optional. Is there a case when it's null?
Same for isSwapV2?

Reply:
you right, fixed